google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/ManagedVmCloudConfigurable.java

```diff
@@ -153,9 +148,20 @@ public boolean isModified() {
 
   @Override
   public void apply() throws ConfigurationException {
-    configuration.setCloudProjectName(projectSelector.getText());
-    configuration.setCloudSdkExecutablePath(
-        CloudSdkUtil.toExecutablePath(cloudSdkDirectoryField.getText()));
+    if (StringUtil.isEmpty(cloudSdkDirectoryField.getText())
```

Comment:
Depends on the context from which this panel is loaded (it can appear in the IJ Cloud Settings menu, or as a popup from within the App Server panel). Here are the two variations (in the first case its that red block area near the bottom):

![image](https://cloud.githubusercontent.com/assets/1735744/13330630/a2f1dd58-dbc7-11e5-836d-ff96fe0b0a01.png)

![image](https://cloud.githubusercontent.com/assets/1735744/13330654/cb070408-dbc7-11e5-8eaf-041b03830e62.png)